packages/ckeditor5-build-classic/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ckeditor5-custom-build",
-  "version": "35.3.1",
+  "version": "35.3.0",
```

Comment:
Might want to use 35.3.2 then? As you will have a problem with caching in the artifactory

Reply:
Good point

Reply:
And @hilhorstt i'm just changing this line to 35.3.2, right?